CHANGELOG.md

```diff
@@ -70,6 +70,7 @@
 ### Fuse.Scripting
 - Restore accidentally broken NativeEvent.RaiseAsync() API.
 - Rolled back NativeProperty constructor API to the pre-1.6 state. Code updated to 1.6 needs to be rolled back as well.
+- Previous changes resulted in a breaking change we would rather avoid so we reintroduce `Invoke(Action ..)` so we can take it through the usual deprecation cycle.
```

Comment:
I guess something should be mentioned in the changelog about this not making it into 1.7?

Reply:
yeah I was thinking to do that in another commit, but I wouldn't mind if it were added in this set of patches.

Reply:
@yupferris: Done.